pkgs/development/python-modules/nipype/neurdflib.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, isodate
+, html5lib
+, SPARQLWrapper
+, networkx
+, nose
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "neurdflib";
+  version = "5.0.0";
+
+  src = fetchPypi {
+    inherit pname;
+    version = "5.0.0.post1"; # This is a packaging release. No code has been modified.
```

Comment:
Even if it is, what's the harm in using `5.0.0.post1` as the version? It would make auto-upgrades possible.